samples/bluetooth/bap_unicast_server/src/main.c

```diff
@@ -666,8 +667,8 @@ static int set_location(void)
 
 	if (IS_ENABLED(CONFIG_BT_PAC_SRC_LOC)) {
 		err = bt_pacs_set_location(BT_AUDIO_DIR_SOURCE,
-					   (BT_AUDIO_LOCATION_FRONT_LEFT |
-					    BT_AUDIO_LOCATION_FRONT_RIGHT));
+						(BT_AUDIO_LOCATION_FRONT_LEFT |
+						BT_AUDIO_LOCATION_FRONT_RIGHT));
```

Comment:
Indentation is off here, should match the starting parenthesis (basically just revert this change)